src/SolidStateDetectors.jl

```diff
@@ -29,7 +29,18 @@ using YAML
 
 include("ConstructiveSolidGeometry/ConstructiveSolidGeometry.jl")
 using .ConstructiveSolidGeometry
-using .ConstructiveSolidGeometry: CylindricalPoint, CartesianPoint
+using .ConstructiveSolidGeometry: 
+            CylindricalPoint, CartesianPoint, AbstractCoordinatePoint, _convert_point,
+            CartesianVector, CylindricalVector, AbstractCoordinateVector,
+            Cartesian, Cylindrical, AbstractCoordinateSystem, CoordinateSystemType,
+            CartesianTicksTuple, CylindricalTicksTuple,
+            Geometry, AbstractGeometry,
+            get_decomposed_volumes,
+            get_decomposed_surfaces, AbstractSurfacePrimitive,
+            geom_round, geom_sigdigits, geom_atol_zero
+            
+import .ConstructiveSolidGeometry: sample, sample_surface
+export CartesianPoint, CartesianVector, CylindricalPoint

```

Comment:
Should we export more things from the CSG?
For example: `Cartesian` and `Cylindrical`?

Reply:
I don't think `Cartesian` and `Cylindrical` would be used by the normal user